lib/event.rb

```diff
@@ -0,0 +1,12 @@
+class Event
+  attr_reader :name, :crafts, :person
+  def initialize(name, crafts, person)
+    @name = name
+    @crafts = [crafts]
+    @person= [person]
+  end
```

Comment:
You are close here. You are sending the craft and person objects in as arrays already, so you don't need the extra brackets on lines 5 and 6. Your return values ends up being  an array within an array `[[#<Craft:0x00007fbc499188d8 @name="knitting", @supplies_required={:yarn=>20, :scissors=>1, :knitting_needles=>2}>]]` rather than a single`[#<Craft:0x00007fbc499188d8 @name="knitting", @supplies_required={:yarn=>20, :scissors=>1, :knitting_needles=>2}>]`  as specified in the interaction pattern.